flask_utils/decorators.py

```diff
@@ -173,7 +173,6 @@ def _check_type(value: Any, expected_type: Type, allow_empty: bool = False, curr
 
 
 def validate_params(
-    parameters: Dict[Any, Any],
     allow_empty: bool = False,
```

Comment:
Do we still need `allow_empty` since we have the `Optional` types ? I use to use `allow_empty` for empty strings for example but I don't know how `typing.Optional` handles that

Reply:
Sorry I don't quite understand :P What does the allow_empty option do?
It means to count empty strings when validating as strings?

Reply:
Yeah, the idea was that sometimes I wanted to allow empty strings to be passed to a flask route :)

Reply:
I mean it should work the same as it did before :P
I didn't change the `_check_type` function, the actual format of the data in `paramaters` hasn't changed because `get_type_hints` returns a dict in the same format as it was before.

Reply:
Okay no worries. I'm writing a bunch of tests anyway to see if what you did does what I want haha, so I'll let you know soon !

Reply:
Sounds good 👍